docs/getting-started.md

```diff
@@ -212,11 +212,12 @@ are in different subnets, but does not encapsulate when the source and the
 destination Nodes are in the same subnet. Refer to [this guide](noencap-hybrid-modes.md)
 to learn how to configure Antrea with `NoEncap` or `Hybrid` mode.
 
-### Octant UI
+### Antrea web UI
 
-Antrea ships with an Octant UI plugin which can show runtime information of Antrea
-components and perform Antrea Traceflow operations. Refer to [this guide](octant-plugin-installation.md)
-to learn how to install Octant and the Antrea plugin.
+Antrea comes with a web UI, which can show runtime information of Antrea
+components and perform Antrea Traceflow operations. Please refer to the [Antrea
+UI repository](https://github.com/antrea-io/antrea-ui) for installation
```

Comment:
Could you check and fix the UI helm chart issue https://github.com/antrea-io/antrea-ui/issues/11?
antrea-ui chart is not in antrea repo yet.

Reply:
There is nothing to fix. There is a development version of the chart available, but the `helm install` command is not supposed to work until v0.1.0 is released.